modal/stub.py

```diff
@@ -677,12 +677,25 @@ def webhook(
 
         To learn how to use Modal with popular web frameworks, see the
         [guide on web endpoints](https://modal.com/docs/guide/webhooks).
+
+        All webhook requests have a 150s maximum request time for the HTTP request itself. However, the underlying functions can
+        run for longer.
```

Comment:
```suggestion
        run for longer and return results to the caller on completion.
```

Reply:
The suggestion is in the direction of reassuring the reader that the function calls aren't 'orphaned' from the HTTP client in any way.